docs/tutorials/substreams/getting-started.experimental.md

```diff
@@ -0,0 +1,318 @@
+---
+title: 'Getting Started'
+
+order: -9999
+outline: [0,4]
+---
+
+# Substreams Quickstart
+
+The goal of this document is to get a development environment set up in as little time as possible.
+
+Here is a quick introduction to [Substreams](https://substreams.streamingfast.io/).
+
+## Prerequisites
+
+### Installing `helm`
+
+-   [Helm for Windows](https://helm.sh/docs/intro/install/#from-chocolatey-windows).
+-   [Helm for Linux](https://helm.sh/docs/intro/install/#from-script).
+-   [Helm for Mac](https://helm.sh/docs/intro/install/#from-homebrew-macos).
+
+Make sure `helm` is installed properly:
+
+```sh
+helm version
+```
+
+### Installing `minikube`
+
+Make sure you already install `docker`, if not you can follow our [Docker Quick-start](../docker/getting-started.md).
+
+-   [Minikube for Windows/Linux/Mac](https://minikube.sigs.k8s.io/docs/start/).
+
+Make sure `minikube` can start properly:
+
+```sh
+minikube version
+```
+```sh
+minikube version: v1.32.0
+commit: 8220a6eb95f0a4d75f7f2d7b14cef975f050512d
+```
+
+### Installing `kubectl`
+
```

Comment:
Should explain why both `minikube` and `kubectl` are necessary, how either of the two will be used, etc.

Reply:
Add a short explanation on top.
Since the purpose of this tutorial is to let user set this up with ease, if users want to know more about the tool, they can make their own research